prow/cmd/webhook-server/main.go

```diff
@@ -172,15 +171,22 @@ func main() {
 		logrus.WithError(err).Fatal("could not get necessary ca secret files", err)
 	}
 	configAgent, err := o.config.ConfigAgent()
+	cfg := configAgent.Config()
 	if err != nil {
 		logrus.WithError(err).Fatal("could not create config agent")
 	}
+	wa := &webhookAgent{
+		storage:  o.storage,
+		statuses: statuses,
+		def:      cfg.Plank.GuessDefaultDecorationConfig("", ""),
```

Comment:
defining default here will only get us the bare min defaults for each. Can we add the the plank config to the webhook agent and call Plank.mergeDefaultDecorationConfig when we have received the prowjob and know the repo and/or cluster